src/App.tsx

```diff
@@ -1,7 +1,35 @@
 import React from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+type State = {
+  pressedKey: string;
+};
+
+export class App extends React.Component<{}, State> {
+  state = {
+    pressedKey: '',
```

Comment:
It would be better if you put null here. It's mean that you don't have value now
```suggestion
    pressedKey: null,
```